nvm.sh

```diff
@@ -1999,6 +2015,10 @@ nvm_supports_source_options() {
   [ "_$(echo 'echo $1' | . /dev/stdin yes 2> /dev/null)" = "_yes" ]
 }
 
+nvm_supports_xz() {
+  command which xz 2>&1 >/dev/null && nvm_is_merged_node_version "$1"
```

Comment:
Sure. This is bit of a mixed bag though. We messed up during release phase, so `xz` tarballs for darwin is `2.3.2` and up whereas linux/src is `1.0.0` and up. Go with the safe option of `2.3.2` and up?